.github/workflows/ci.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
       - lint
       - build
     steps:
+      - name: Uninstall Postgresql
+        run: |
+          sudo apt --purge remove postgresql*
```

Comment:
Workaround for 24.04, which has preinstalled PG

Reply:
Do you mean GH runner 24.04?

Reply:
Yes.